src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -19,7 +19,8 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives wi
   }
 
   val rawlsDAO:RawlsDAO = new HttpRawlsDAO
-  val searchDAO:SearchDAO = new ElasticSearchDAO
+  val searchDAO:SearchDAO = new ElasticSearchDAO(FireCloudConfig.ElasticSearch.servers, FireCloudConfig.ElasticSearch.indexName)
+  searchDAO.initIndex
```

Comment:
I'm currently seeing the log messages for initIndex appearing twice in the logs. I'm researching that ...